docs/content/specs-defs/includes/shared/shared/non-functional/SNFR2.md

```diff
@@ -85,3 +85,32 @@ resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2021-12
 ```
 
 {{% /expand %}}
+
+##### Skipping Deployments (**SHOULD NOT**)
+
+Tests are part of the documentation and ensure thorough testing of the module and parameters. However, there are situation where an actual deployment can't be performed. One could be, that the deployment is too big (the Azure Resource Manager has a [limit of 4MB](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/best-practices) for a single deployment) or only one instance of a service can be deployed to a subscription. To still keep a test for reference and documentation, a file named `.e2eignore` can be placed inside a test folder to prevent the test case being deployed by the CI.
```

Comment:
The first sentene is a bit wacky. Do you mean the tests are part of the AVM documentation? Or that they're injected as usage examples into a module's readme?